draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3666,10 +3670,13 @@ Server: example-server/0.1
    <section title="Invalidate" anchor="examples-invalidate2">
    <t>
    Below is an example of a CI/T "invalidate" trigger creation. 
-   This trigger instructs the dCDN to revalidate the content at "https://www.example.com/a/index.html", 
-   as well as any metadata and content whose URLs are prefixed by "https://metadata.example.com/a/b/" using
-   case-insensitive matching, and "https://www.example.com/a/b/" using case-sensitive matching, respectively.
+   This trigger instructs the dCDN to revalidate:
    </t>
+   <list style="symbols">
+   <t>the metadata objects with URLs prefixed by "https://metadata.example.com/a/b/" using case-insensitive matching</t>
+   <t>the content object with "https://www.example.com/a/index.html" URL"</t>
```

Comment:
This bullet item has an unbalanced quotation mark.  I would suggest maybe rewording it to:
> a single content object identified by the URL "https://www.example.com/a/index.html"